packages/dd-trace/test/config.spec.js

```diff
@@ -92,6 +93,7 @@ describe('Config', () => {
     const tags = {
       'foo': 'bar'
     }
+    const mockDisabledIntegrations = ['express']
```

Comment:
Small nit, but it would be slightly cleaner to use the same name as the option.

Reply:
👍 updated